phpunit/class-registration-test.php

```diff
@@ -13,31 +13,37 @@ class Registration_Test extends WP_UnitTestCase {
 	function tearDown() {
 		parent::tearDown();
 
-		foreach ( WP_Block_Type_Registry::get_instance()->get_all_registered() as $name => $block_type ) {
-			WP_Block_Type_Registry::get_instance()->unregister( $name );
+		$registry = WP_Block_Type_Registry::get_instance();
+
+		foreach ( array( 'core/dummy' ) as $name ) {
```

Comment:
I think we may as well remove these `foreach` since we're only iterating over an array with one element.